nautobot/extras/models/customfields.py

```diff
@@ -589,6 +585,16 @@ def to_form_field(
 
         return field
 
+    def to_filter_field(self, *args, **kwargs):
```

Comment:
This method is named `to_filter_field` and says that it's returning a filter field, but the code appears to actually be returning a form field. Can we please clarify this?

Reply:
Yes, A filter field is a form field used for filtering.
Maybe it should be called `to_filter_form_field`?

Reply:
`to_filter_form_field` would be clearer, yes.